codec-agnostic-rtp-payload-format.md

```diff
@@ -43,27 +43,37 @@ informative:
   RFC6464:
   RFC6465:
   RFC6904:
+  RFC6184:
   SFrame:
+    target: https://tools.ietf.org/html/draft-omara-sframe
+    title: Secure Frame (SFrame)
+  WebRTCInsertableStreams:
+    target: https://w3c.github.io/webrtc-insertable-streams
+    title: WebRTC Insertable Media using Streams
 
 --- abstract
 
-
+WebRTC Media Chains usually rely on piping encoder output directly to packetizers. Media packetization formats often support a specific codec format and optimize RTP packets generation accordingly.
```

Comment:
It should not be WebRTC specific, lets change WebRTC by RTP and just mention WebRTC in the contest of insertable streams.

Reply:
SFrame is kind of WebRTC specific. If there is no SFU trying to select packets, SRTP might be sufficient for instance.
I think we should clearly state that the main application scope is WebRTC.
I'll remove it there but will keep it below

Reply:
In the Sframe charter we state that is is a solution for encrypting realtime media for RTP.

https://datatracker.ietf.org/doc/charter-ietf-sframe/

Reducing the scope to WebRTC will cause a lot of discussions while providing no benefit (it it works for RTP in general, it works for webrtc)

Reply:
I tried to take a stab at it. Let me know what you think in the new version.